specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -3535,6 +3535,125 @@
+            "$ref": "./examples/IpAddressAggregates.json"
+          }
+        },
+        "operationId": "list_ipAddressAggregatesByService",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "skiptoken",
+            "in": "query",
+            "description": "A continuationtoken value returned in paginated result to load different pages.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The list of IP addresses.",
+            "schema": {
+              "$ref": "#/definitions/IpAddressAggregates"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.ADHybridHealthService/services/{serviceName}/ipAddressAggregateSettings": {
+      "get": {
+        "tags": [
+          "Reports"
+        ],
+        "description": "Gets the ipAddressAggregateSettings.",
+        "x-ms-examples": {
+          "get_ipAddressAggregateSettings": {
+            "$ref": "./examples/IpAddressAggregateSettings.json"
+          }
+        },
+        "operationId": "get_ipAddressAggregateSettings",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ip AddressAggregate Settings.",
+            "schema": {
+              "$ref": "#/definitions/IpAddressAggregateSetting"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Configuration"
+        ],
+        "description": "Updates the ipAddressAggregateSettings alert thresholds.",
+        "x-ms-examples": {
+          "update_ipAddressAggregateSettings": {
+            "$ref": "./examples/IpAddressAggregateSettingsUpdate.json"
+          }
+        },
+        "operationId": "update_ipAddressAggregateSettings",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "ipAddressAggregateSetting",
+            "in": "body",
+            "description": "The ipAddressAggregateSetting object.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/IpAddressAggregateSetting"
```

Comment:
Could up update the **definitions** where has Ip in them to be IP? These will be generated as model classes. We try to do some normalization, but it would be good to keep clean in the swagger.

Reply:
Ok, so I'll change IpAddressAggregateSetting to IPAddressAggregateSetting

Reply:
Fixed